runelite-client/src/main/java/net/runelite/client/ui/PluginPanel.java

```diff
@@ -45,7 +45,7 @@
 	@Getter(AccessLevel.PROTECTED)
 	private final JScrollPane scrollPane;
 
-	@Getter(AccessLevel.PACKAGE)
+	@Getter(AccessLevel.PROTECTED)
```

Comment:
Don't do this. Disable scrolling instead and create your own scrollbar

Reply:
After much deliberation I think I've sussed a clever way to avoid using PluginPanel's wrappedPanel.
Let me know if you think the new implementation could be done better.